src/ExposureHistory/ExposureDatumDetail.tsx

```diff
@@ -42,8 +42,9 @@ const PossibleExposureDetail = ({
   const exposureDurationText = DateTimeUtils.durationToString(duration)
   const navigation = useNavigation()
   const { t } = useTranslation()
-  const displayNextSteps =
-    env.DISPLAY_SELF_ASSESSMENT === "true" || env.AUTHORITY_ADVICE_URL
+  const displayNextSteps = Boolean(
+    env.DISPLAY_SELF_ASSESSMENT === "true" || env.AUTHORITY_ADVICE_URL,
+  )
```

Comment:
Maybe this wants to leave in a configuration context, I have a feeling that this won't be the single "exception" to the flow we will have to handle